Makefile

```diff
@@ -70,21 +70,20 @@ release_quay: test build
 
 remove_network: stop_postgres
 	@echo "Removing the docker network (if it exists)"
-	-docker network rm $(COMPOSED_NETWORK)
+	-docker network rm $(COMPOSED_NETWORK) || true
 
 start_postgres: create_network stop_postgres
 	@echo "Starting postgres"
 	@docker run --rm -d --name $(POSTGRES_HOST) \
         -e POSTGRES_USER=$(POSTGRES_USER) \
         -e POSTGRES_PASSWORD=$(POSTGRES_PASSWORD) \
         -e POSTGRES_DB=$(POSTGRES_DB) \
-        -p 5432:5432 \
```

Comment:
By removing this we lose the ability for users to iterate quickly with by using `pytest` from the command line (vs. running re-building the test containers and running the whole test suite via `make test` every time). Since we are running into conflicts on this port perhaps we use a different port instead (54321, 55432, etc.). Whatever we choose, we need to update the db_config fixture in the conftest.py as well to match.

Reply:
fixed with another "are you in docker?" check